modules/dists/SparseBlockDist.chpl

```diff
@@ -328,24 +315,31 @@ class LocSparseBlockDom {
 // locArr: a non-distributed array of local array classes
 // myLocArr: optimized reference to here's local array class (or nil)
 //
-class SparseBlockArr: BaseArr {
-  type eltType;
-  param rank: int;
-  type idxType;
+class SparseBlockArr: BaseSparseArr {
   param stridable: bool;
   type sparseLayoutType = DefaultDist;
-  var dom; //: SparseBlockDom(rank, idxType, stridable);
-  var locArr: [dom.dist.targetLocDom] LocSparseBlockArr(eltType, rank, idxType,
-      stridable, sparseLayoutType);
-  var myLocArr: LocSparseBlockArr(eltType, rank, idxType, stridable, 
+
+  // child class' fields initializers will not have dom set up correctly. domain
+  // of locArr should be dom.dist.targetLocDom
```

Comment:
Is this comment still accurate? Are you talking about BaseSparseArr.dom. But that doesn't make since since these are different domains. Not sure what the comment is saying.